### PR TITLE
feature(playback-rate): Support playback rate control in player

### PR DESCRIPTION
**Pull request by steffentchr**

### CODE REVIEW

File: src/video-display/video-display.js [hunk 1]
```diff
@@ -622,6 +622,7 @@ Player.provide('video-display',
       });
       Player.setter('playbackRate', function(rate) {
         $this.video && $this.video.setPlaybackRate(rate);
+        Player.fire('player:video:playbackratechange');
```
Comment (AbrakaBarna) [inline src/video-display/video-display.js:625]: I got a `$this.video.setPlaybackRate is not a function` error in the console when trying to do this, but I'm assuming you're aware of this

**Review by AbrakaBarna — COMMENTED** [review]
can you also look at the files I changed?

File: src/video-display/video-display.js (hunk 1 above)
Comment (steffentchr) [reply src/video-display/video-display.js:625]: @AbrakaBarna You may need to pull on you eingebaut repo, https://github.com/23/eingebaut/commit/aefb9b78d800e36f96fcde6f2097df8e54c71eb3

**Comment by steffentchr** [comment]
@AbrakaBarna Am assuming this one got in there by mistake?
Screenshot 2023-03-23 at 23 43 27

Also, did you manage to run the player with updated dependencies, making sure it worked?

File: src/video-display/video-display.js (hunk 1 above)
Comment (AbrakaBarna) [reply src/video-display/video-display.js:625]: yes, that helped

**Review by AbrakaBarna — APPROVED** [review]

**Pull request merged**